### ISSUE TITLE
okta-react: ImplicitCallback fails in React.StrictMode because setState is called on unmounted component

### ISSUE THREAD

**Issue by mschaeffner** [okta/okta-oidc-js#178]
**How to reproduce:**

* go to react test application and update dependency so that it uses React >= 16.3.0
*  in file `App.js` wrap the `ImplicitCallback` route with a `React.StrictMode` (see https://reactjs.org/docs/strict-mode.html)

change from
```
<Route path='/implicit/callback' component={ImplicitCallback} />
```
to
```
<React.StrictMode>
  <Route path='/implicit/callback' component={ImplicitCallback} />
</React.StrictMode>
```
* Run the application and login with username and password
* The callback is performed and the URL will be http://localhost:3000/implicit/callback
* The application crashes with error: `AuthSdkError: Unable to parse a token from the url`
* The javascript console shows the warning:
```
Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op.

Please check the code for the ImplicitCallback component.
```

**Comment by ripplegupta03** [comment]
I am also facing the same issue, what is the above fix planned to release?
Is there any other workaround for now?

**Comment by mschaeffner** [comment]
The only workaround I know is not using `React.StrictMode` 🙈 
I also like to see this getting merge asap.

**Comment by robertjd** [comment]
This change is now available in version 1.0.2 on npm. Thanks again @mschaeffner !
